fili-sql/src/main/java/com/yahoo/bard/webservice/sql/DruidQueryToSqlConverter.java

```diff
@@ -929,6 +939,55 @@ private void sqlAggregationHelper(
         return allGroupBys;
     }
 
+    /**
+     * Find all word positions in string.
+     * @param textString a string to search into.
+     * @param word a target word.
+     * @return an array of indexes of all positions word was found in the textString.
```

Comment:
style issues @params two spaces after param name.
1 line of whitespace between params and returns/throws.